index.d.ts

```diff
@@ -845,6 +847,7 @@ declare namespace Eris {
   }
   interface AdvancedMessageContent {
     allowedMentions?: AllowedMentions;
+    attachments?: {id: string, filename: string};
```

Comment:
this should be made into its own interface or separated out into different lines, it looks out of place as-is

that should also be a semicolon, not a comma
```suggestion
    attachments?: {
        id: string;
        filename: string
    };
```

Reply:
I didn't realize Attachment was already an interface lol